spec/friendly_shipping/services/usps/parse_address_validation_response_spec.rb

```diff
@@ -26,13 +26,13 @@
     end
   end
 
-  # context 'with a no-candidates request' do
-  #   let(:response_body) { File.open(File.join(gem_root, 'spec', 'fixtures', 'ups', 'address_validation_no_candidates_response.xml')).read }
+  context 'with a no-candidates request' do
+    let(:response_body) { File.open(File.join(gem_root, 'spec', 'fixtures', 'usps', 'address_validation_no_candidates_response.xml')).read }
 
-  #   it { is_expected.to be_failure }
+    it { is_expected.to be_failure }
 
-  #   it 'has the correct error message' do
-  #     expect(subject.failure.to_s).to eq("Address is probably invalid. No similar valid addresses found.")
-  #   end
-  # end
+    it 'has the correct error message' do
+      expect(subject.failure.to_s).to eq("-2147219401: Address Not Found.")
+    end
+  end
```

Comment:
Hmm, can we pend the spec in the previous commit instead of commenting it?

Reply:
I've moved the full spec to the previous commit. I just didn't have the fixture by the time I imported the code.